internal/yggdrasil/yggdrasil.go

```diff
@@ -365,6 +365,8 @@ func (h *Handler) PostDataMessageForDevice(ctx context.Context, params yggdrasil
 		if err == nil {
 
 			if !IsOwnDevice(ctx, deviceID) {
+				authKeyVal, _ := ctx.Value(AuthzKey).(string)
+				logger.V(0).Info("Device tries to re-register with an invalid certificate", "certcn", authKeyVal)
```

Comment:
@masayag should I add metrics for this situation, registration with a renew-certificate, and an invalid one?

Reply:
That's a nice suggestion.
I'm thinking about expanding this not only to failed cert-renewal but for any violation of `IsOwnDevice` .
In that case, not only the registration API can be accessed using an invalid cert, but any other unauthorized access to the service.

Reply:
:+1: I'll open a issue for that